### DISCUSSION TITLE
Error: parseNode in parser/model/ParserHtml.ts

### DISCUSSION THREAD

**Discussion by andcmatias** [GrapesJS/grapesjs#5289] [Q&A]
I don't know if it's something I'm doing wrong or a bug.

When inserting a component I am getting the error "Cannot create property 'tagName' on number '0'".

When analyzing the code of version 0.21.4 I found where the error is generated.
On line 177, model = obj is always set, it turns out that on line 166 obj is always set obj = compType.model.isComponent(node, opts);
And in some situations 0 is returned.

And on line 185 model.tagName the error is generated, because the model is set to 0.

I fixed the error by changing line 177 to:
if(typeof obj === "object")) model = obj;

**Comment by artf** [comment]
I don't think we're able to provide help here without a reproducible demo...
